delpicture.php

```diff
@@ -52,8 +52,8 @@
  */
 
 if ($xCube) {
-    if (!$xoopsGTicket->check(true, 'token')) {
-        redirect_header($_SERVER['HTTP_REFERER'], 3, $xoopsGTicket->getErrors());
+    if (!$GLOBALS['xoopsSecurity']->check(true, $_REQUEST['token'])) {
```

Comment:
With this change, we don't need a special case for $xCube. Both if and else do the same thing, only the form field name changes. If we are using xoopsSecurity in all cases, taking the default name in all cases would make things cleaner.